### PR TITLE
Mosaic table view

### PR DESCRIPTION
**Pull request by chowington**
Blocks https://github.com/VEuPathDB/web-eda/pull/277

Waiting on decision regarding tabs vs showing both views side-by-side
Update: decision made to go with tabs

### CODE REVIEW

File: src/components/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import { Tabs as MUITabs, Tab, AppBar } from '@material-ui/core';
+import { TabPanel, TabContext } from '@material-ui/lab';
+
+interface Props {
+  items: Array<[string, React.ReactElement]>;
```
Comment (chowington) [inline src/components/Tabs.tsx:6]: Maybe should clarify this items prop?

File: src/components/Tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+export default function Tabs(props: Props) {
+  const [activeTab, setActiveTab] = useState('0');
```
Comment (chowington) [inline src/components/Tabs.tsx:10]: If we choose to keep these tabs, might want to make this component stateless/controlled

**Review by bobular — APPROVED** [review]

**Comment by dmfalke** [comment]
@chowington please update the description so that issues are not automatically linked. It causes issues to be closed automatically, which doesn't work with the new project flow.

My recommendation is to add a section like this, at the bottom of your description:

----

### Related issues
- https://github.com/VEuPathDB/web-eda/issues/237 (closes)
- https://github.com/VEuPathDB/web-eda/pull/277 (blocks)

File: src/components/Tabs.tsx (hunk 1 above)
Comment (chowington) [reply src/components/Tabs.tsx]: Done

File: src/components/Tabs.tsx (hunk 2 above)
Comment (chowington) [reply src/components/Tabs.tsx:13]: Will save for a later ticket

**Comment by chowington** [comment]
Changing to 'ready for review' based on Steve's suggestion to go ahead and merge this now

**Review by jtlong3rd — APPROVED** [review]

**Review by dmfalke — APPROVED** [review]

**Review by moontrip — APPROVED** [review]

**Pull request merged**